warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__scheduled_trips_in_tu_feed.sql

```diff
@@ -0,0 +1,63 @@
+WITH
+
+services_guideline_index AS (
+    SELECT * FROM {{ ref('int_gtfs_quality__services_guideline_index') }}
+),
+
+fct_observed_trips AS (
+    SELECT * FROM {{ ref('fct_observed_trips') }}
+),
+
+fct_daily_scheduled_trips AS (
+    SELECT * FROM {{ ref('fct_daily_scheduled_trips') }}
+),
+
+dim_provider_gtfs_data AS (
+    SELECT * FROM {{ ref('dim_provider_gtfs_data') }}
+),
+
+joined AS (
+    SELECT
+       idx.date,
+       idx.service_key,
+       COUNT(scheduled_trips.trip_id) AS scheduled_trips,
+       COUNT(observed_trips.trip_id) AS observed_trips,
+    FROM services_guideline_index AS idx
+
+    LEFT JOIN dim_provider_gtfs_data AS quartet
+    ON idx.service_key = quartet.service_key
+    AND TIMESTAMP(idx.date) BETWEEN quartet._valid_from AND quartet._valid_to
```

Comment:
Same comments as #2228 here -- I think we need a filter on `guidelines_assessed` here on `dim_provider_gtfs_data` on both joins